src/main/scala/me/shadaj/scalapy/numpy/NDArray.scala

```diff
@@ -30,6 +30,11 @@ class NDArray[T](val value: PyValue)(implicit reader: Reader[T]) extends py.Obje
 
   def shape: Seq[Int] = origDynamic.shape.as[Seq[Int]]
 
+  override def slice(from: Int, until: Int): NDArray[T] = {
```

Comment:
should this be a method of NDArray instance?
in python a call is `array[slice(0,100), ...]`
(the idea is to allow for multi dimensional slicing)

Reply:
Hmmm IIUC the same remark applies to `length`, `apply` and `iterator` - they're all methods of Scala `Seq` that do not directly correspond to methods of the Python class. But still, since we aim at ease of use of this class from Scala code, I think adding a bit of Scala-like methods is okay (or actually even desirable)

Reply:
And yes, we could also add a `slice` method that does not implement stuff from `scala.Seq`... but I don't think that's necessary for us at this point